docs/api-reference/layers/editable-geojson-layer.md

```diff
@@ -354,3 +302,36 @@ The pointer moved, regardless of whether the pointer is down, up, and whether or
 * `groundCoords` (Array): `[lng, lat]` ground coordinates where the pointer is now.
 * `isDragging` (Boolean): `true` if the pointer is moving but it is considered a drag, in this case you likely want to handle `onDragging`
 * `pointerDownPicks` (Array): An array containing [deck.gl Picking Info Objects](https://uber.github.io/deck.gl/#/documentation/developer-guide/adding-interactivity?section=what-can-be-picked-) for all objects that were under the pointer when it went down, or `null` if pointer is moving without pointer down.
+
+
+## Override SubLayers
+
+Three default sublayers make up the composite `EditableGeoJsonLayer`:
+
+1. Layer with `id: geojson` renders the geometry. Default: `GeoJsonLayer`
+2. Layer with `id: edit-handles` renders the edit handles. Default: `ScatterplotLayer`
+3. Layer with `id: draw` renders the geometry during a `draw` mode which has not been "committed" yet. This includes the last segment of a line during `drawLineString` and the fill during `drawPolygon`.  Default: `GeoJsonLayer`
+
+All three sublayer defaults can be overridden with custom or experimental layers. Define a prop object that contains a `Layer` class object, `props`, and `accessors`. The key should be the `id` of the layer to override. For instance:
+
+```javascript
+{
+  'geojson': {
+    Layer: OutlinedGeoJsonLayer,
+    props: {
```

Comment:
Why the separation and nesting into `accessors` and `props`?

It seems like a gratuitous difference compared to base deck.gl API. 

Anything we can do to avoid it?